python/mxnet/_numpy_op_doc.py

```diff
@@ -176,3 +176,229 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         `axis` is not None or `a` is a 1-d array.
     """
     pass
+
+
+def _np_amax(a, axis=None, out=None):
+    r"""
+    amax(a, axis=None, out=None, keepdims=_Null, initial=_Null)
+
+    Return the maximum of an array or maximum along an axis.
+
+    Parameters
+    ----------
+    a : ndarray
+        Input data.
+        Python native iterables not supported.
+
+    axis : None or int or tuple of ints, optional
+        Axis or axes along which to operate. By default, flattened input is
+        used.
+
+        Negative indices not supported.
+
+    out : ndarray, optional
```

Comment:
Is `out` really the 3rd argument of amax for our version? I believe all MXNet operators put `out` in the end, not in the middle.

Reply:
fixed